### PR TITLE
move frame

### PR DESCRIPTION
**Pull request by senchpimy**
Move video with "," and "." keys like in youtube

### CODE REVIEW

File: youtube/templates/watch.html [hunk 1]
```diff
@@ -678,6 +678,20 @@ <h2 class="title">{{ title }}</h2>
 
     <script src="/youtube.com/static/js/av-merge.js"></script>
     <script src="/youtube.com/static/js/watch.js"></script>
+    <script>
+    var vido = document.getElementById('cuurent-vid'),
+    frameTime = 1 / 30;
```
Comment (user234683) [inline youtube/templates/watch.html:683]: Different videos on YouTube have different framerates (e.g. 60 fps)

File: youtube/templates/watch.html [hunk 2]
```diff
@@ -473,7 +473,7 @@
     {% else %}
         <div id="video-container">
             <div id="video-container-inner">
-                <video controls autofocus class="video" height="{{ video_height }}px">
+                <video controls autofocus class="video" id="cuurent-vid" height="{{ video_height }}px">
```
Comment (user234683) [inline youtube/templates/watch.html:476]: No need to add this id to the css

File: youtube/templates/watch.html [hunk 3]
```diff
@@ -678,6 +678,20 @@ <h2 class="title">{{ title }}</h2>
 
     <script src="/youtube.com/static/js/av-merge.js"></script>
     <script src="/youtube.com/static/js/watch.js"></script>
+    <script>
+    var vido = document.getElementById('cuurent-vid'),
+    frameTime = 1 / 30;
+window.addEventListener('keypress', function (evt) {
```
Comment (user234683) [inline youtube/templates/watch.html:684]: indent properly

File: youtube/templates/watch.html [hunk 4]
```diff
@@ -678,6 +678,20 @@ <h2 class="title">{{ title }}</h2>
 
     <script src="/youtube.com/static/js/av-merge.js"></script>
     <script src="/youtube.com/static/js/watch.js"></script>
+    <script>
+    var vido = document.getElementById('cuurent-vid'),
```
Comment (user234683) [inline youtube/templates/watch.html:682]: video, not vido. And use document.querySelector('video') to get the video

**Review by user234683 — CHANGES_REQUESTED** [review]
I suggest looking on stackoverflow for a more standard way to advance by one frame, if it exists. If not, you'll need to get the framerate of the selected format from the other code in youtube local

File: youtube/templates/watch.html (hunk 1 above)
Comment (senchpimy) [reply youtube/templates/watch.html:683]: > I suggest looking on stackoverflow for a more standard way to advance by one frame, if it exists. If not, you'll need to get the framerate of the selected format from the other code in youtube local
Thats the stackoverflow answer

Comment (senchpimy) [reply youtube/templates/watch.html]: It could be fixed getting the fps from the selector?

**Pull request closed without merging**